### ISSUE TITLE
404 Error when trying voice cloning with Vall-E-X

### ISSUE THREAD

**Issue by IJustWantToSetTheWorldOnFire** [mudler/LocalAI#1732]
**LocalAI version:**
localai/localai:v2.8.2-cublas-cuda12-ffmpeg

**Environment, CPU architecture, OS, and Version:**
Windows 11 Home 23H2, CPU : 12th Gen Intel(R) Core(TM) i9-12900H   2.90 GHz, GPU : Nvidia RTX 3060 Laptop, 32 GB of RAM

**Describe the bug**
The voice cloning starts generating, but is halted by a 404 error and produces an empty output file.

**To Reproduce**
Launched a container with the following command :
docker run -e DEBUG=true -ti -p 8080:8080 --mount type=bind,source=c:\voices\models,destination=/models,bind-propagation=shared --mount type=bind,source=c:\voices\audio,destination=/tmp/generated/audio,bind-propagation=shared --gpus all localai/localai:v2.8.2-cublas-cuda12-ffmpeg vall-e-x

logged as root in the container via the command su -

Copied the necessary files in the right place with the following command :
cp /models/cloned.yaml /build/models && cp /models/DunmerM.wav /build/models

Ran voice cloning with the following command : 
curl http://localhost:8080/tts -H "Content-Type: application/json" -d '{"backend": "vall-e-x", "model": "cloned-voice", "input": "Bonjour, ceci est un essai"}' > /models/output.wav

**Expected behavior**
A generated wav file in /models based on the cloned voice saying "Bonjour, ceci est un essai"

**Logs**
8:10AM DBG Request for model: cloned-voice
8:10AM INF Loading model 'cloned-voice' with backend vall-e-x
8:10AM DBG Loading model in memory from file: /build/models/cloned-voice
8:10AM DBG Loading Model cloned-voice with gRPC (file: /build/models/cloned-voice) (backend: vall-e-x): {backendString:vall-e-x model:cloned-voice threads:0 assetDir:/tmp/localai/backend_data context:{emptyCtx:{}} gRPCOptions:0xc000224800 externalBackends:map[autogptq:/build/backend/python/autogptq/run.sh bark:/build/backend/python/bark/run.sh coqui:/build/backend/python/coqui/run.sh diffusers:/build/backend/python/diffusers/run.sh exllama:/build/backend/python/exllama/run.sh exllama2:/build/backend/python/exllama2/run.sh huggingface-embeddings:/build/backend/python/sentencetransformers/run.sh mamba:/build/backend/python/mamba/run.sh petals:/build/backend/python/petals/run.sh sentencetransformers:/build/backend/python/sentencetransformers/run.sh transformers:/build/backend/python/transformers/run.sh transformers-musicgen:/build/backend/python/transformers-musicgen/run.sh vall-e-x:/build/backend/python/vall-e-x/run.sh vllm:/build/backend/python/vllm/run.sh] grpcAttempts:20 grpcAttemptsDelay:2 singleActiveBackend:false parallelRequests:false}
8:10AM DBG Loading external backend: /build/backend/python/vall-e-x/run.sh
8:10AM DBG Loading GRPC Process: /build/backend/python/vall-e-x/run.sh
8:10AM DBG GRPC Service for cloned-voice will be running at: '127.0.0.1:32829'
8:10AM DBG GRPC Service state dir: /tmp/go-processmanager1384383809
8:10AM DBG GRPC Service Started
8:10AM DBG GRPC(cloned-voice-127.0.0.1:32829): stderr /opt/conda/envs/transformers/lib/python3.11/site-packages/torch/nn/utils/weight_norm.py:30: UserWarning: torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.
8:10AM DBG GRPC(cloned-voice-127.0.0.1:32829): stderr   warnings.warn("torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.")
8:10AM DBG GRPC(cloned-voice-127.0.0.1:32829): stderr Downloading: "https://dl.fbaipublicfiles.com/encodec/v0/encodec_24khz-d7cc33bc.th" to /root/.cache/torch/hub/checkpoints/encodec_24khz-d7cc33bc.th
100%|██████████| 88.9M/88.9M [00:04<00:00, 22.4MB/s]r
8:10AM DBG GRPC(cloned-voice-127.0.0.1:32829): stderr Server started. Listening on: 127.0.0.1:32829
[127.0.0.1]:41628 200 - GET /readyz
8:10AM DBG GRPC Service Ready
8:10AM DBG GRPC: Loading model with options: {state:{NoUnkeyedLiterals:{} DoNotCompare:[] DoNotCopy:[] atomicMessageInfo:<nil>} sizeCache:0 unknownFields:[] Model:cloned-voice ContextSize:0 Seed:0 NBatch:512 F16Memory:false MLock:false MMap:false VocabOnly:false LowVRAM:false Embeddings:false NUMA:false NGPULayers:0 MainGPU: TensorSplit: Threads:4 LibrarySearchPath: RopeFreqBase:0 RopeFreqScale:0 RMSNormEps:0 NGQA:0 ModelFile:/build/models/cloned-voice Device: UseTriton:false ModelBaseName: UseFastTokenizer:false PipelineType: SchedulerType: CUDA:false CFGScale:0 IMG2IMG:false CLIPModel: CLIPSubfolder: CLIPSkip:0 ControlNet: Tokenizer: LoraBase: LoraAdapter: LoraScale:0 NoMulMatQ:false DraftModel: AudioPath: Quantization: MMProj: RopeScaling: YarnExtFactor:0 YarnAttnFactor:0 YarnBetaFast:0 YarnBetaSlow:0 Type:}
8:10AM DBG GRPC(cloned-voice-127.0.0.1:32829): stderr Preparing models, please wait
[127.0.0.1]:43708 200 - GET /readyz
config.yaml: 100%|██████████| 503/503 [00:00<00:00, 2.76MB/s]
pytorch_model.bin: 100%|██████████| 40.4M/40.4M [00:01<00:00, 21.4MB/s]
8:11AM DBG GRPC(cloned-voice-127.0.0.1:32829): stderr text: "Bonjour, ceci est un essai"
8:11AM DBG GRPC(cloned-voice-127.0.0.1:32829): stderr model: "/build/models/cloned-voice"
8:11AM DBG GRPC(cloned-voice-127.0.0.1:32829): stderr dst: "/tmp/generated/audio/piper.wav"
8:11AM DBG GRPC(cloned-voice-127.0.0.1:32829): stderr
[127.0.0.1]:37670 404 - POST /tts
[127.0.0.1]:41666 200 - GET /readyz

Thanks in advance.

**Comment by TwinFinz** [comment]
the generated audio file is NOT in the models. "/tmp/generated/audio/piper.wav". this is also working in the latest image, Altho if you try to generate anything long it tends to just respond random sounds (this issue specifically is a model issue not backend)
so i suggest updating to the latest image and testing again.
